container_images/test-manager/Dockerfile

```diff
@@ -0,0 +1,40 @@
+# Copyright 2020 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+FROM golang as builder
+
+WORKDIR /go/src/
+COPY . .
+
+WORKDIR ./test_manager
+RUN go get
+RUN CGO_ENABLED=0 GOOS=linux go build -v -o /test_manager
+
+WORKDIR ./test_wrapper
+RUN go get
+RUN CGO_ENABLED=0 GOOS=linux go build -v -o /test_wrapper
+
+WORKDIR ../test_suites/oslogin
+RUN CGO_ENABLED=0 GOOS=linux go test -c -o /oslogin.test
+
+WORKDIR ../shutdown_scripts
+RUN CGO_ENABLED=0 GOOS=linux go test -c -o /shutdown_scripts.test
+
+FROM alpine
+COPY --from=builder /test_manager /
+COPY --from=builder /test_wrapper /
+COPY --from=builder /shutdown_scripts.test /
+COPY --from=builder /oslogin.test /
```

Comment:
lets copy recursively so we don't need to write each item down. it will be painful if people add a new test package and don't realize they have to update the dockerfile as well

Reply:
Gotcha. Then we need a output folder instead a root /